docs/docs/backup_howto.md

```diff
@@ -0,0 +1,235 @@
+# Backup strategy guide
+
+This guide explains how to design and implement a backup strategy in Xen Orchestra.  
+
+Instead of a simple list of questions and answers, it walks you through **key decisions** and **best practices** before, during, and after setting up backups.
+
+---
+
+## Glossary
+
+This part explains the terminology of backup types and features.
+
+- **Backup sequence**: A feature that allows you to chain multiple backup jobs to run one after the other, automatically. 
+- [**File restore**](./backups#restore-a-file): A feature that allows you to restore individual files from a VM backup without restoring the full VM.
+- [**Full backup**](./full_backups): Copies the entire VM each time, regardless of previous backups. 
+- [**Full replication**](./full_replication): Creates a complete replica of a VM on another host or pool.  
+- [**Incremental backup**](./incremental_backups): Saves only the changes since the last backup, reducing storage needs. 
```

Comment:
```suggestion
- [**Incremental backup**](./incremental_backups): Transfer and stores only the changes since the last backup to a backup repositories , reducing storage and network needs. The first transfer transfers the VM completly.
```